authentication-saml/src/main/java/com/synopsys/integration/alert/authentication/saml/database/SAMLMetaDataModeConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.synopsys.integration.alert.authentication.saml.database;
+
+import com.synopsys.integration.alert.authentication.saml.model.SAMLMetadataMode;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.stream.Stream;
+
+@Converter(autoApply = true)
+public class SAMLMetaDataModeConverter implements AttributeConverter<SAMLMetadataMode, Integer> {
```

Comment:
We should probably do this for any other enums that we save to the database.

Reply:
Created https://jira-sig.internal.synopsys.com/browse/IALERT-3320